filter/series_by_tag_pattern_index.go

```diff
@@ -54,7 +59,7 @@ func NewSeriesByTagPatternIndex(
 		}
 
 		if patternMatching.nameTagValue == "" {
-			withoutStrictNameTagPatternMatchers[pattern] = patternMatching.matchingHandler
+			withoutStrictNameTagPatternMatchers = append(withoutStrictNameTagPatternMatchers, patternAndHandler{pattern, patternMatching.matchingHandler})
```

Comment:
Придирка (можно не исправлять)
```suggestion
			withoutStrictNameTagPatternMatchers = append(
			       withoutStrictNameTagPatternMatchers, 
			       patternAndHandler{
			           pattern: pattern, 
			           handler: patternMatching.matchingHandler,
			           })
```

Reply:
Согласен, исправил